<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task_RunAlgo</class>
 <widget class="QWidget" name="task_RunAlgo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="evaluatePanel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbIsEvalPerf">
       <property name="text">
        <string>Evaluate Perfomance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browse_gtDir">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Selecting Groundtruth Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtGtDir">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browse_resultDir">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Save Eval as</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtResultDir">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Default save output to ~/../My Documents/xResults</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="saveOutputPanel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="saveImageFormat">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Images Format</string>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="saveJPG">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>jpg</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="saveBMP">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>bmp</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="savePNG">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>png</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbIsSaveImage">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Export output Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSaveImageOutputPath">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>E:\QTWS\dataset</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget_5</zorder>
   <zorder>saveImageFormat</zorder>
  </widget>
  <widget class="QPushButton" name="btnProcess">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PROCESS</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="paramPanel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>351</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameters :</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="btnLoadDefaultConfig">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Load Default Parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetParam" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Algorithms Selection</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="selAlgo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Select Method below or select this to Extract frame as images</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Basic methods-FrameDifferenceBGS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Basic methods-DPMeanBGS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Single gaussian based methods-LBSimpleGaussian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiple gaussians based methods-DPGrimsonGMMBGS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Type 2 Fuzzy based methods-T2FGMM_UM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiple features based methods-DPTextureBGS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiple features based methods-SJN_MultiCueBGS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiple features based methods-LOBSTERBGS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiple features based methods-SuBSENSEBGS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Non parametric methods-KDE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Non parametric methods-IndependentMultimodalBGS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Non parametric methods-PBAS</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="stepB">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pPB">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Play/Pause</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
